R/utilities-data-set.R

```diff
@@ -109,9 +106,19 @@ dataSetToDataFrame <- function(dataSets) {
     df[[name]] <- col
   }
 
-  return(dplyr::as_tibble(df))
+  # consistently return a (classical) data frame
+  return(as.data.frame(df))
```

Comment:
Why do you use `as.data.frame` here? Isn't `df` already created as a classical df?

Reply:
Reverted.